### PR TITLE
Fix/frontpage

### PR DESCRIPTION
**Pull request by Roverr**

### CODE REVIEW

File: src/components/Architecture/Architecture.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Container } from 'reactstrap';
+
```
Comment (bartaroland) [inline src/components/Architecture/Architecture.jsx]: what is this?

File: src/components/Architecture/Elements/PicRow.jsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { Col } from 'reactstrap';
```
Comment (bartaroland) [inline src/components/Architecture/Elements/PicRow.jsx]: Empty line?

File: src/components/Mentorship/Mentorship.jsx [hunk 3]
```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import { Container, Col, Row } from 'reactstrap';
 import SubHeader from '../SubHeader';
 import SubContent from './Elements/SubContent';
-import Interested from './Elements/Interested';
 import { RowStyle } from './Styled';
 import PicWithText from './Elements/PicWithText';
 import PicRow from './Elements/PicRow';
+import Info from '../Info/Info';
+
```
Comment (bartaroland) [inline src/components/Mentorship/Mentorship.jsx]: Empty line?

Comment (Roverr) [reply src/components/Mentorship/Mentorship.jsx]: Separates the imports from the images, makes it more readable

**Pull request merged**